### ISSUE TITLE
Incorrect PYTHON_HOME Path

### ISSUE THREAD

**Issue by rainman110** [mabrarov/build-scripts#51]
The current windows_dev docker image install Python 3.9.1 under c:\Python37:
https://github.com/mabrarov/build-scripts/blob/d3c42ed0dfaea2381b207bff6ed32faa8a890c87/docker/windows-dev/src/Dockerfile#L29

This might cause confusion, if python 3.7 is installed via choco.

The installation path should be changed to c:\Python39.

**Comment by mabrarov** [comment]
Hi @rainman110,

Thank you for the feedback!

I believe that more generic names:

```
    PYTHON3_HOME="C:\Python3" \
    ...
    PYTHON2_HOME="C:\Python2"
```

can be a better choice. 

What do you think?

**Comment by rainman110** [comment]
More generic names to don't conflict with other python installation routines (e.g. using chocolatey), or alternatively use python39 for a python 3.9 install, python37 for a python 3.7 install.

My problem was, that I needed python 3.7 and chocolatey installed python 3.7 into c:\Python37. Since there was already a python 3.9 installation in the c:\python37 directory, python was not usable anymore after installation.

**Comment by mabrarov** [comment]
Fixed in #52
